Ryujinx/Ui/SwitchSettings.cs

```diff
@@ -0,0 +1,431 @@
+            _fsLogSpinAdjustment.Value           = SwitchConfig.FsGlobalAccessLogMode;
+
+            _gameDirsBox.AppendColumn("", new CellRendererText(), "text", 0);
+            _gameDirsBoxStore  = new ListStore(typeof(string));
+            _gameDirsBox.Model  = _gameDirsBoxStore;
+            foreach (string gameDir in SwitchConfig.GameDirs)
+            {
+                _gameDirsBoxStore.AppendValues(gameDir);
+            }
+
+            if (_custThemeToggle.Active == false)
+            {
+                _custThemePath.Sensitive      = false;
+                _custThemePathLabel.Sensitive = false;
+                _browseThemePath.Sensitive    = false;
+            }
+
+            _logPath.Buffer.Text = System.IO.Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "Ryujinx.log");
+
+            _listeningForKeypress = false;
+        }
+
+        //Events
+        private void Button_Pressed(object obj, EventArgs args, ToggleButton Button)
+        {
+            if (_listeningForKeypress == false)
+            {
+                KeyPressEvent += On_KeyPress;
+                _listeningForKeypress = true;
+
+                void On_KeyPress(object Obj, KeyPressEventArgs KeyPressed)
+                {
+                    string key    = KeyPressed.Event.Key.ToString();
+                    string capKey = key.First().ToString().ToUpper() + key.Substring(1);
+
+                    if (Enum.IsDefined(typeof(OpenTK.Input.Key), capKey)) { Button.Label = capKey;                }
+                    else if (GdkToOpenTKInput.ContainsKey(key))           { Button.Label = GdkToOpenTKInput[key]; }
+                    else                                                  { Button.Label = "Space";               }
+
+                    Button.SetStateFlags(0, true);
+                    KeyPressEvent -= On_KeyPress;
+                    _listeningForKeypress = false;
+                }
+            }
+            else { Button.SetStateFlags(0, true); }
+        }
+
+        private void AddDir_Pressed(object obj, EventArgs args)
+        {
+            if (Directory.Exists(_addGameDirBox.Buffer.Text)) { _gameDirsBoxStore.AppendValues(_addGameDirBox.Buffer.Text); }
+
+            _addDir.SetStateFlags(0, true);
+        }
+
+        private void BrowseDir_Pressed(object obj, EventArgs args)
+        {
+            FileChooserDialog fc = new FileChooserDialog("Choose the game directory to add to the list", this, FileChooserAction.SelectFolder, "Cancel", ResponseType.Cancel, "Add", ResponseType.Accept);
+
+            if (fc.Run() == (int)ResponseType.Accept)
+            {
+                _gameDirsBoxStore.AppendValues(fc.Filename);
+            }
+
+            fc.Destroy();
+
+            _browseDir.SetStateFlags(0, true);
+        }
+
+        private void RemoveDir_Pressed(object obj, EventArgs args)
+        {
+            TreeSelection selection = _gameDirsBox.Selection;
+
+            selection.GetSelected(out TreeIter treeiter);
```

Comment:
Remove new line before and add a new line after.

```suggestion
            selection.GetSelected(out TreeIter treeIter);
```